media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="An image viewer and organizer built with Qt and Exiv2"
+DESCRIPTION="Features
+* Support for common image formats and GIF animation
+* Supports tagging images, and filtering images by tags (IPTC)
+* Browse thumbnails recursively down a folder tree
+* Dynamic thumbnails loading
+* Image transformation and color manipulation
+* Display image information and metadata
+* Does not depend on any desktop environment."
+HOMEPAGE="https://github.com/oferkv/phototonic"
+COPYRIGHT="2015-2018 Over Kashayov
+	2015-2018 Christopher roy
+	2015-2018 Krzysztof Pyrkosz
+	2015-2018 Roman Chistokhodov
+	2015-2018 Thomas Lübking
+	2015-2018 Tung Le"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/oferkv/phototonic/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a8b2dbc81750efabb65a4732b1704641afd4a4f772ef9c90817871ca2a52f5ef"
+SOURCE_FILENAME="phototonic-$portVersion.tar.gz"
+PATCHES="phototonic-$portVersion.patchset"
+ADDITIONAL_FILES="phototonic.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+USER_SETTINGS_FILES="
+	settings/PhotoTonic directory
+	"
+PROVIDES="
+	phototonic$secondaryArchSuffix = $portVersion
+	app:PhotoTonic = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexiv2$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
```

Comment:
@mazbrili, could you try this command:
```
objdump -x /system/apps/PhotoTonic | grep NEEDED | grep -i libQt5 | sort
```
and see if you get the same output I'm getting?
```
  NEEDED               libQt5Core.so.5
  NEEDED               libQt5Gui.so.5
  NEEDED               libQt5Widgets.so.5
```
If you do, then I guess we can (or should?) remove some lines here (and in `BUILD_REQUIRES` too):
```diff
	lib:libQt5Core$secondaryArchSuffix
	lib:libQt5Gui$secondaryArchSuffix
-	lib:libQt5Network$secondaryArchSuffix
-	lib:libQt5Sql$secondaryArchSuffix
-	lib:libQt5Svg$secondaryArchSuffix
 	lib:libQt5Widgets$secondaryArchSuffix
```

Reply:
sorry @fbrosson  for long reply .. yes same with you..  ok edited

Reply:
No problem! Having contributions is great!